README.md

```diff
@@ -220,6 +220,35 @@ envsubst < deployment/private-installer.yaml | kubectl apply -f -
 
 The AWS Secrets Manager and Config Provider provides compatibility for legacy applications that access secrets as mounted files in the pod. Security conscious applications should use the native AWS APIs to fetch secrets and optionally cache them in memory rather than storing them in the file system.
 
+### Client-Side Rate-Limitting to Kubernetes API server
+
+To mount each secret on each pod, the AWS CSI provider lookups the region of the pod and the role ARN associated with the service account by calling the K8s APIs. You can increase the value of qps and burst if you notice the provider is throttled by client-side limit to the API server.
```

Comment:
It's probably better to just write out "Kubernetes" instead of shortening to "K8s" to avoid confusion for readers.

Reply:
Check with @ecraw-amzn

Reply:
Revised in new commit.